docs/kyma/04-07-local-installation-deprecated.md

```diff
@@ -0,0 +1,185 @@
+      <summary>
+      Linux
+      </summary>
+
+      ```
+      ./cmd/run.sh --password {USER_PASSWORD} --vm-driver virtualbox
+      ```
+      </details>
+    </div>
+
+8. By default, the Kyma installation is a background process, which allows you to perform other tasks in the terminal window. Nevertheless, you can track the progress of the installation by running the `is-installed.sh` script. It is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
+
+  ```
+  ./scripts/is-installed.sh
+  ```
+  >**TIP:** If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script. If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
+
+9. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password you set using the `--password` flag.
+
+10. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+
+Read [this](#installation-reinstall-kyma) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
+To learn how to test Kyma, see [this](#details-testing-kyma) document.
+
+## Stop and restart Kyma without reinstalling
+
+Use the `minikube.sh` script to restart the Minikube cluster without reinstalling Kyma. Follow these steps to stop and restart your cluster:
+
+1. Stop the Minikube cluster with Kyma installed. Run:
+  ```
+  minikube stop
+  ```
+2. Restart the cluster without reinstalling Kyma. Run:
+    <div tabs>
+      <details>
+      <summary>
+      MacOS
+      </summary>
+
+      ```
+      ./scripts/minikube.sh --domain "kyma.local" --vm-driver hyperkit
+      ```
+      </details>
+      <details>
+      <summary>
+      Linux
+      </summary>
+
+      ```
+      ./scripts/minikube.sh --domain "kyma.local" --vm-driver virtualbox
+      ```
+      </details>
+    </div>
+
+The script discovers that a Minikube cluster is initialized and asks if you want to delete it. Answering `no` causes the script to start the Minikube cluster and restarts all of the previously installed components. Even though this procedure takes some time, it is faster than a clean installation as you don't download all of the required Docker images.
+
+To verify that the restart is successful, run this command and check if all Pods have the `RUNNING` status:
+
+```
+kubectl get pods --all-namespaces
+```
+
+## Enable Horizontal Pod Autoscaler (HPA)
```

Comment:
What about this section? It looks important. Perhaps it should be copied to the main document for local installation?

Reply:
if you mean "nable Horizontal Pod Autoscaler (HPA)", this is enabled OOB with CLI and nothing todo additionally to enable it